components/VanthSwiper.js

```diff
@@ -135,86 +169,25 @@ export default function VanthSwiper({ autoplay }) {
 					},
 				}}
 			>
-				<SwiperSlide>
-					<Image
-						src={bunnyImg}
-						alt='Bunny Vanth'
-						layout='fill'
-						objectFit='cover'
-						objectPosition='center'
-					/>
-				</SwiperSlide>
-				<SwiperSlide>
-					<Image
-						src={catImg}
-						alt='Cat Vanth'
-						layout='fill'
-						objectFit='cover'
-						objectPosition='center'
-					/>
-				</SwiperSlide>
-				<SwiperSlide>
-					<Image
-						src={wolfImg}
-						alt='Wolf Vanth'
-						layout='fill'
-						objectFit='cover'
-						objectPosition='center'
-					/>
-				</SwiperSlide>
-				<SwiperSlide>
-					<Image
-						src={mothImg}
-						alt='Moth Vanth'
-						layout='fill'
-						objectFit='cover'
-						objectPosition='center'
-					/>
-				</SwiperSlide>
-				<SwiperSlide>
-					<Image
-						src={owlImg}
-						alt='Owl Vanth'
-						layout='fill'
-						objectFit='cover'
-						objectPosition='center'
-					/>
-				</SwiperSlide>
-				<SwiperSlide>
-					<Image
-						src={deerImg}
-						alt='Deer Vanth'
-						layout='fill'
-						objectFit='cover'
-						objectPosition='center'
-					/>
-				</SwiperSlide>
-				<SwiperSlide>
-					<Image
-						src={foxImg}
-						alt='Fox Vanth'
-						layout='fill'
-						objectFit='cover'
-						objectPosition='center'
-					/>
-				</SwiperSlide>
-				<SwiperSlide>
-					<Image
-						src={ravenImg}
-						alt='Raven Vanth'
-						layout='fill'
-						objectFit='cover'
-						objectPosition='center'
-					/>
-				</SwiperSlide>
-				{/* <SwiperSlide>
-                    <Image
-                      src={vanthImg}
-                      alt='Vanth Symbol'
-                      width='1080'
-                      height='1080'
-                    />
-                  </SwiperSlide> */}
+				{VanthImages.map((VanthImage) => (
+					<SwiperSlide key={VanthImage}>
```

Comment:
I fetch your PRs to check out the changes and I received an error in the console. 

```
Warning: Encountered two children with the same key, `.0:$[object Object]`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```
Must not like the object itself as the key. I read that having unique keys like id's or using hashing functions will work. 
Would something like this be fine:
```
const VanthImages = [
	{
		id: 1,
		src: bunnyImg,
		alt: 'Bunny Vanth',
	},
	{
		id: 2,
		src: catImg,
		alt: 'Cat Vanth',
	},
	//and so on for the rest of the images
```
and 
```
				{VanthImages.map((VanthImage) => (
					<SwiperSlide key={VanthImage.id}>
						<Image
						//and so on with the rest 
```
I built it with like that and the error went away.

Reply:
Great catch, I totally missed that error. I'm going to use the alt text for the key so I don't need to add extra properties to the VanthImages objects, it seems to fix that error.